tests/FunctionalJavascript/AddCorporateFormTest.php

```diff
@@ -0,0 +1,174 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_contact_forms\FunctionalJavascript;
+
+use Drupal\FunctionalJavascriptTests\WebDriverTestBase;
+
+/**
+ * Test Thirdparty settings on contact forms.
+ */
+class AddCorporateFormTest extends WebDriverTestBase {
+
+  /**
+   * An test user with permission to create contact forms.
+   *
+   * @var \Drupal\user\UserInterface
+   */
+  protected $testuser;
```

Comment:
`$testUser`. Also, any reason why this is set as a class property?

Reply:
Class property removed and use a local variable in `setup()` function